frontend/kesaseteli/employer/public/locales/sv/common.json

```diff
@@ -107,7 +107,7 @@
         "employment_work_hours": "Tehdyt työtunnit SWE",
         "employment_salary_paid": "Maksettu palkka SWE",
         "hired_without_voucher_assessment": "Olitko palkannut nuoren ilman kesäseteliä? SWE",
-        "termsAndConditions": "Olen lukenut palvelun käyttöehdot ja hyväksyn ne. SWE"
+        "termsAndConditions": "SWE Olen lukenut palvelun <1>käyttöehdot</1> ja hyväksyn ne"
```

Comment:
is it so that for hs project you use SWE abbreviation and for benefit and others used SV?